spec/gitlab_merge_request_hook_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+require 'gitlab_merge_request_hook'
+
+describe GitlabMergeRequestHook do
+  let(:changes)         { "123456 789012 refs/heads/#{branch_name}" }
+  let(:config)          { GitlabConfig.new }
+  let(:repository_path) { "/home/git/repositories" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.